src/doc/rustdoc/src/unstable-features.md

```diff
@@ -209,6 +209,22 @@ some consideration for their stability, and names that end in a number). Giving
 `rustdoc` will disable this sorting and instead make it print the items in the order they appear in
 the source.
 
+### `--show-type-layout`: add a section to each type's docs describing its memory layout
+
+Using this flag looks like this:
+
+```bash
+$ rustdoc src/lib.rs -Z unstable-options --show-type-layout
```

Comment:
Let's not show the prompt, it's different on Windows.
```suggestion
rustdoc src/lib.rs -Z unstable-options --show-type-layout
```
(And maybe change `bash` to `console` above.)

Reply:
I agree, but I wanted to be consistent with the other sections, which use `bash` and `$ `.

Reply:
Ah ok, that makes sense. I didn't realize they were using this format already.